evap/staff/forms.py

```diff
@@ -95,6 +95,79 @@ def validate_unique(self):
             self._update_errors(e)
 
 
+class SingleResultForm(forms.ModelForm, BootstrapMixin):
+    last_modified_time_2 = forms.DateTimeField(label=_("Last modified"), required=False, localize=True)
+    last_modified_user_2 = forms.CharField(label=_("Last modified by"), required=False)
+    event_date = forms.DateField(label=_("Event date"), localize=True)
+    responsible = forms.ChoiceField(choices=[(u.username, u) for u in UserProfile.objects.order_by('username')], label=_("Responsible"))
```

Comment:
the [ModelChoiceField](https://docs.djangoproject.com/en/1.8/ref/forms/fields/#fields-which-handle-relationships) looks more appropriate and probably solves this.